src/SFA.DAS.EAS.Employer_Account.Database/StoredProcedures/GetTheRequiredRdsStatistics.sql

```diff
@@ -0,0 +1,28 @@
+﻿CREATE PROCEDURE [employer_account].[GetTheRequiredRdsStatistics]
+
+AS
+SELECT 
+(
+	SELECT COUNT(Id) 
+	FROM [employer_account].[Account]
+) AS TotalAccounts,
+(
+	SELECT COUNT(Id)
+	FROM [employer_account].[LegalEntity]
+	WHERE status = 'active'
+) AS TotalLegalEntities,
+(
+	SELECT DISTINCT COUNT(Ref)
```

Comment:
I don't think this will give you what you need. This should work though `SELECT COUNT(DISTINCT Ref) FROM [employer_account].[Paye]`.